source/installation/preparation.rst

```diff
@@ -32,6 +32,12 @@ Le processus d'installation ne vous permettra pas d'installer Galette s'il ne lu
 * |folder| `data/tempimages`,
 * |folder| `data/templates_c`
 
+Ou pour faire ça en une ligne de commande :
+
+.. code-block:: bash
+
+   # chmod u+w galette/galette/{config,data/attachments,data/cache,data/exports,data/files,data/imports,data/logs,data/photos,data/tempimages,data/templates_c}
```

Comment:
In facts, `galette/{config,data/}` would be enough ;) (and maybe better since plugins may use `data` directory)